system-tests/tests/src/test/java/org/eclipse/edc/test/system/remote/FileTransferAsClientSimulation.java

```diff
@@ -51,6 +48,11 @@ public FileTransferAsClientSimulation() {
     }
 
     private static String getFromEnv(String env) {
-        return Objects.requireNonNull(StringUtils.trimToNull(System.getenv(env)), env);
```

Comment:
My question here is: did we need this `trimToNull` at all? I'd say that `System.getenv` with a `isBlank` check would already do the job. The variables are set in the `minikube-test.sh` and I can't see a case where they have leading-trailing whitespaces to trim or be an empty string.
I would change the implementation of this method to 
`Objects.requireNonNull(System.getenv(env), env)`

Reply:
Though bailing out fast on empty/invalid value may help to find the cause on future bug, I think it should be enough to check the null value here as you suggest. I will update the code.